### ISSUE TITLE
[CLI] Add support for flag to keep answers value as string

### ISSUE THREAD

**Issue by PennyScissors** [rancher/rancher#33072]
Issue: https://github.com/rancher/rancher/issues/26088
Related PR: https://github.com/rancher/rancher/pull/31837
Add support in the rancher CLI for the new "answersSetString" API field so that answers can be forced set to string type.

**Comment by thehejik** [comment]
Tested on `rancher/rancher:master-head 4d41301`

TL;DR works as expected

* downloaded rancher-cli from https://releases.rancher.com/cli2/v2.4.12-rc2/rancher-linux-amd64-v2.4.12-rc2.tar.gz
* logged in to my rancher server by `rancher login https://myrancher --token <atokenhash>` and switched context to my downstream RKE2 cluster and Default project
* Tried to install redis by using rancher cli with following testing flags:
```
./rancher app install --set-string usePassword=false --set-string foo=bar  --set cluster.enabled=false --set usePassword=false redis redis-helm
```
* Checked API json for the installed redis app:

![rancher-cli-answersSetString](https://user-images.githubusercontent.com/12828077/125455261-0ed628ba-69c0-41a1-8e12-1fe27f9e52ff.png)

From the screenshot you can see that the key-pairs from rancher-cli command were correctly assigned to the coresponding groups of answers `answers/answersSetString`.

Close.
